meta-aos-rcar-gen4-dom0/recipes-core/runx/runx_git.bbappend

```diff
@@ -0,0 +1,54 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+SRCREV_runx = "0c7edb3453398d7a0c594ce026c9c1e93c2541cc"
+
+SRC_URI = "\
+	  git://github.com/lf-edge/runx;nobranch=1;name=runx \
+	  file://0001-files-create-Update-create-script-to-start-unikernel.patch \
+	  "
+
+SRC_URI[md5sum] = "ce9b2d974d27408a61c53a30d3f98fb9"
+SRC_URI[sha256sum] = "bf338980b1670bca287f9994b7441c2361907635879169c64ae78364efc5f491"
+
+PV = "v1.1-git${SRCREV_runx}"
+REQUIRED_DISTRO_FEATURES_remove = "vmsep"
+
+DEPENDS_remove = "busybox"
```

Comment:
All `DPENDS_remove` can be combined in one var

Reply:
fixed